spec/std/http/web_socket_spec.cr

```diff
@@ -28,6 +28,15 @@ private def assert_packet(packet, opcode, size, final = false)
   packet.final.should eq(final)
 end
 
+private class MalformerHandler
```

Comment:
typo: `Malformer` -> `Malformed`

Reply:
It's not a typo. It's a handler that malforms the response 🙂